variants/STM32WBxx/WB55R(C-E-G)V/variant_STEAM32_WB55RG.h

```diff
@@ -0,0 +1,233 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2021, STMicroelectronics
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+#pragma once
+
+/*----------------------------------------------------------------------------
+ *        STM32 pins number
+ *----------------------------------------------------------------------------*/
+
+// micro:bit Edge connector
+#define PC4                     PIN_A1
+#define PA5                     PIN_A3
+#define PC5                     PIN_A5
+#define PA2                     PIN_A0
+#define PA4                     PIN_A2
+#define PA7                     5
+#define PC3                     6
+#define PA9                     7
+#define PA15                    8
+#define PC2                     9
+#define PA6                     PIN_A4
+#define PA8                     11
+#define PC6                     12
+#define PB13                    13
+#define PB14                    14
+#define PB15                    15
+#define PE4                     16
+#define PC0                     17
+#define PC1                     18
+
+// Not on connectors
+#define PB2                     19
+#define PD0                     20
+#define PB8                     21
+#define PB9                     22
+#define PC13                    23
+#define PB12                    24
+#define PB0                     25
+#define PD1                     26
+#define PB6                     27
+#define PB7                     28
+#define PC10                    29
+#define PH3                     30
+#define PC11                    31
+#define PC12                    32
+#define PA0                     33
+#define PA3                     34
+#define PA10                    35
+#define PA12                    36
+#define PB1                     37
+#define PB10                    38
+#define PB11                    39
+#define PA11                    40
+#define PB4                     41
+#define PB5                     42
+#define PA1                     43
+
+#define NUM_DIGITAL_PINS        44
+#define NUM_ANALOG_INPUTS       6
+
+// GPIO expander
+// Upper pad
+#define GPIO1                   44 // P28
+#define GPIO2                   45 // P29
+#define GPIO3                   46 // P24
+#define GPIO4                   47 // P25
+
+// 4 directions switches
+#define GPIO5                   48
+#define GPIO6                   49
+#define GPIO7                   50
+#define GPIO8                   51
```

Comment:
How do you plan to use those GPIO expander aliases?

Reply:
outdated as no more value

Reply:
I removed this definition. 

At the begining, I want to investigate a solution to hide the use of an IO Expander with for example a macro for `digitalRead` and `digitalWrite` but it's probably confusing on several use case of the Pin outside basic LED blinking. I'll make samples to show how to use the IO expander, it will be more interesting.